src/Login.js

```diff
@@ -60,18 +61,30 @@ class Login extends React.Component {
   render() {
     return (
       <div className = "Login" >
-        <header> Welcome to PP Friends</header>
+        <div className = "top">
+        <svg viewBox="0 0 500 200">
+          <path id="curve" fill = "transparent" d="M73.2,200.6c4-6.1,65.5-96.8,178.6-95.6c111.3,1.2,170.8,90.3,175.1,97" />
+          <text width="500">
+          <textPath href="#curve">
+            Making friendly connections.
+          </textPath>
+            </text>
+        </svg>
+        <img src="ppFriendsLogo.png"></img>
+        </div>
+        
+        <h1> Purely Platonic</h1>
```

Comment:
Are we calling it "Purely Platonic" or "Purely Platonic Friends"?

Reply:
I think purely platonic sounds good, but like when they match its like youre purely platonic friends ( like facebook friends?), not sure we can discuss in standup.